src/components/ListView.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import ListCard from '../components/ListCard';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = theme => ({
+  container: {
+    backgroundColor: 'grey',
+    display: 'flex',
+    alignItems: 'center',
+    flexDirection: 'column'
+  }
+});
+
+class ListView extends React.Component {
+  constructor(props) {
```

Comment:
We can pull this out of the constructor so it's just `state={};` at the top of the class.